src/peergos/server/net/SocialHandler.java

```diff
@@ -0,0 +1,84 @@
+package peergos.server.net;
+
+import com.sun.net.httpserver.*;
+import peergos.server.*;
+import peergos.server.social.*;
+import peergos.server.util.*;
+import peergos.shared.crypto.hash.*;
+import peergos.shared.social.*;
+import peergos.shared.util.*;
+
+import java.io.*;
+import java.util.*;
+import java.util.function.*;
+import java.util.logging.*;
+
+public class SocialHandler implements HttpHandler {
```

Comment:
class docstring plz.

Reply:
what is this used by?

Reply:
This is used by the Peergos server (UserService)